src/test/java/gov/nasa/pds/validate/ValidationIntegrationTests.java

```diff
@@ -760,6 +743,57 @@ void testGithub173() {
         }
     }
     
+    @Test
+    void testGithub206() {
+        try {
+            // Setup paths
+            String testPath = Utility.getAbsolutePath(TestConstants.TEST_DATA_DIR + "/github206");
+            String outFilePath = TestConstants.TEST_OUT_DIR;
+            File report = new File(outFilePath + File.separator + "report_github206_1.json");
+
+            // Try with a bad example tha will throw 2 errors
+            String[] args = {
+                    "-r", report.getAbsolutePath(),
+                    "-s", "json",
+                    "-t" , testPath + File.separator + "val9a.xml",
+                    
+                    };
+            this.launcher.processMain(args);
+
+            Gson gson = new Gson();
+            JsonObject reportJson = gson.fromJson(new FileReader(report), JsonObject.class);
+
+            reportJson = gson.fromJson(new FileReader(report), JsonObject.class);
+
+            int count = this.getMessageCount(reportJson, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey());
+
+            assertEquals(count, 1, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey() + " info/error messages expected.");
+
+            // Try with a good example that will throw no errors
+            report = new File(outFilePath + File.separator + "report_github206_2.json");
+            String[] args2 = {
+                    "-r", report.getAbsolutePath(),
+                    "-s", "json",
+                    "-t" , testPath + File.separator + "val9b.xml",
+                    
+                    };
+            this.launcher = new ValidateLauncher();
+            this.launcher.processMain(args2);
+
+            gson = new Gson();
+            reportJson = gson.fromJson(new FileReader(report), JsonObject.class);
+
+            count = this.getMessageCount(reportJson, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey());
+
+            assertEquals(count, 0, ProblemType.FIELD_VALUE_DATA_TYPE_MISMATCH.getKey() + " info/error messages not expected.");
```

Comment:
This test should actually throw an error because this product is described as a Table_Delimited, but the a couple of values contain all spaces for ASCII_Integer. That is invalid. You can only have all spaces for Table_Character. For Table_Delimited, you can wrap an integer with spaces, they just can be all spaces.